bugbug/bug_features.py

```diff
@@ -85,15 +85,20 @@ def __call__(self, bug):
 
 class whiteboard(object):
     def __call__(self, bug):
-        ret = []
 
-        # TODO: Add any [XXX:YYY] that appears in the whiteboard as [XXX: only
+        paren_splits = bug['whiteboard'].lower().split('[')
```

Comment:
Add the comments to explain the various steps, like I did in the previous comment